### PR TITLE
Vet clinic database: querrying and updateing animals table

### PR DESCRIPTION
**Pull request by Kidd254**
final quaerries 2
Final querries 1
Rollback
update
In this pull request I: 
- [ ] Added a column species of type string to my animals table. 
- [ ] Modified my schema.sql file.
- [ ] Inserted data for different animals
- [ ] Added these insertions to my data.sql file.
- [ ] Updated the animals table by setting the species column to 'unspecified' while in a transaction rolled back the change
- [ ] Updated the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- [ ] Updated the animals table by setting the species column to pokemon for all animals that don't have species already set.
- [ ] Commited the transaction.
- [ ] Deleted all records in the animals table, then rolled back
- [ ] Deleted all animals born after Jan 1st, 2022, created a savepoint for the transaction, update all animals' weight to be their weight multiplied by -1, while in a transaction
- [ ] Rolled back to the savepoint
- [ ] Updated all animals' weights that are negative to be their weight multiplied by -1.
- [ ] Commited transaction
- [ ] added all these transactions to your queries.sql file.
- [ ] Wrote querries for different conditions

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -16,3 +16,33 @@ VALUES ('Pikachu', '2021-01-07', 1, FALSE, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', '2017-05-12', 5, TRUE, 11.00);
 
+-- Add a column 'species' of type string to the 'animals' table.
+ALTER TABLE animals ADD COLUMN species VARCHAR(255);
+
```
Comment (DeliceLydia) [inline data.sql]: - [ ] Please move this query in the `schema.sql`

File: queries.sql [hunk 2]
```diff
@@ -8,3 +8,59 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+-- Rollback to the savepoint
+ROLLBACK TO SP1;
+
+-- Update all animals' weights that are negative to be their weight multiplied by -1
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+-- Commit transaction
+COMMIT;
```
Comment (DeliceLydia) [inline queries.sql]: - [ ] Please add these queries
- `UPDATE` the animal's table by setting the species column to unspecified, then roll back the change and remember to `verify that the change was made` and `verify that change was undone`

Comment (DeliceLydia) [reply queries.sql]: - [ ] `UPDATE` the animal's table by setting the species column to digimon for all animals that have a name ending in mon.
- [ ] `UPDATE` the animals table by setting the species column to Pokemon for all animals that don't have species already set.
- [ ] Commit the transaction.

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Kidd254, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Followed git workflow
- Awesome README


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (tresorsawasawa) [inline data.sql]: As mentioned by the previous reviewer and for the project requirements, kindly ensure that everything that creates or change the tables structure should be saved in the `schema.sql` file, as reason the above query to be moved in  `schema.sql` file,

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @Kidd254 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Kidd254 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**